default.nix

```diff
@@ -60,7 +60,19 @@ let
                 else super.lapack.override { isILP64 = true; lapackProvider = super.openblas; };
 
       fftw = self_.fftw.overrideDerivation ( oldAttrs: {
```

Comment:
Let's make this  a separate attribute: `fftw-mpi`.  Compiling mpi into the standard `fftw` means that all applications will depend on MPI even if they do not use it.

I am note sure why I used `overrideDerivation` here. It should be `overrideAttributes`.

Reply:
Added a separate attribute `fftw-mpi`. Interestingly the original `fftw` derivation does not seem to define `overrideAttributes`. It is simply not available.

Reply:
Does it work again if you fix the the above line to:
```
fftw = self_.fftw.overrideAttributes ( oldAttrs: {
```
?